x/concentrated-liquidity/position_test.go

```diff
@@ -2509,3 +2509,49 @@ func (s *KeeperTestSuite) TestCreateFullRangePositionLocked() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestMultipleRanges() {
+	tests := map[string]struct {
+		tickRanges      [][]int64
+		rangeTestParams RangeTestParams
+	}{
+		// The following two tests will fail until the tick rounding bug is fixed (and should pass after):
+		// https://github.com/osmosis-labs/osmosis/issues/5516
+		//
+		// "one min width range": {
+		// 	tickRanges: [][]int64{
+		// 		{0, 100},
+		// 	},
+		// 	rangeTestParams: DefaultRangeTestParams,
+		// },
+		// "two adjacent ranges": {
+		// 	tickRanges: [][]int64{
+		// 		{-10000, 10000},
+		// 		{10000, 20000},
+		// 	},
+		// 	rangeTestParams: DefaultRangeTestParams,
+		// },
+
+		// Both of these lead to underclaiming of fees greater than additive
+		// error tolerance of 1 per token per position. Increasing ticks increases
+		// error disproportionally, while increasing tick range decreases error proportionally.
```

Comment:
Note to reviewer: this is an issue I am still investigating – left the cases here to not block this PR further